ci-operator/config/openshift/osin/openshift-osin-master.yaml

```diff
@@ -0,0 +1,33 @@
+base_images:
+  os:
+    cluster: https://api.ci.openshift.org
+    name: centos
```

Comment:
if possible, I'd base this on the common base image

Reply:
this is the common image for all of apiserver-library-go and library-go, should a completely different config be used here?